### DISCUSSION TITLE
Forma de ordenar txts?

### DISCUSSION THREAD

**Discussion by rinrot** [INGCOM-UNRN-P1/cursada-2023#222] [Preguntas y respuestas] [answered]
Hola! hay alguna manera de usar archivos de texto que estén en una carpeta separada del programa?

**Comment by martinvilu** [comment, marked as answer]
Tenés que indicar la ruta en `open`, por lo menos en Linux, con `/`, si no funciona en Windows, probá con `\` (que debiera de ir como `\\` para que el compilador no lo interprete como una secuencia de escape tipo `\n`)

Con la barra, podés indicar rutas “hacia arriba”, `subdirectorio/archivo`

Pero con `../` podes “bajar” un nivel.

Dada la siguiente estructura:

![imagen](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/56625/74eb87a4-55d1-414e-8aa5-e4c93d77057b)

Con el programa está en `tp5-2023-martinvilu`, podés acceder a otro TP indicando:

`../tp4-2023-martinvilu`

Esto baja a “soluciones” (que contiene a ambas) y vuelve a subir al otro TP.

MUY recomendable, no salgas del directorio del repositorio.
